seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/sink/IcebergSinkWriter.java

```diff
@@ -106,8 +105,7 @@ public static IcebergSinkWriter of(SinkConfig config, CatalogTable catalogTable)
 
     public static IcebergSinkWriter of(
             SinkConfig config, CatalogTable catalogTable, List<IcebergSinkState> states) {
-        IcebergTableLoader icebergTableLoader =
-                IcebergTableLoader.create(config, catalogTable).open();
+        IcebergTableLoader icebergTableLoader = IcebergTableLoader.create(config, catalogTable);
```

Comment:
Why remove the operation of open()?

Reply:
lazy open in https://github.com/apache/seatunnel/blob/e6f92fd79ba4b1bac41c863a7d855548b44c247d/seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/IcebergTableLoader.java#L69